types/plugin/localeData.d.ts

```diff
@@ -6,5 +6,14 @@ export = plugin
 declare module 'dayjs' {
   interface Dayjs {
     localeData(): any
```

Comment:
should we add it here?

Reply:
You mean under interface Dayjs?

If so, I think the exported functions should not sit under the interface since those functions are not instance methods of Dayjs. 

Instead, those functions are like static, not binding to a class.

Reply:
How can typescript knows that weekdaysMin is available on `dayjs` like this dayjs.weekdaysMin()

Reply:
because those functions are exported from module 'dayjs', which has nothing to do with interface Dayjs.

![Screen Shot 2020-10-08 at 10 38 42 am](https://user-images.githubusercontent.com/8860999/95398804-7df35780-0952-11eb-9b4e-32c51b9ba8a5.png)

just like MinMax plugin

Reply:
True, should we take dayjs().localeData().weekdays() into consideration as well?

Reply:
right. I've added localeData types to both dayjs and Dayjs instances.

Now (dayjs or dayjs()).localeData() both have types.